addons/account/models/company.py

```diff
@@ -338,17 +338,17 @@ def _validate_fiscalyear_lock(self, values):
                 ('state', '=', 'draft'),
                 ('date', '<=', values['fiscalyear_lock_date'])])
             if draft_entries:
-                error_msg = _('There are still unposted entries in the period you want to lock. You should either post or delete them.')
+                error_msg = _('There are still draft entries in the period you want to lock. You should either post or delete them.')
                 action_error = {
                     'view_mode': 'tree',
-                    'name': _('Unposted Entries'),
+                    'name': _('Draft Entries'),
```

Comment:
This will break all translations in stable for only little benefit
Should target master instead

Reply:
@Laura-labe would you agree? :slightly_smiling_face:

Reply:
Yes @aliyatastemirova